bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java

```diff
@@ -873,7 +873,9 @@ private long getRequestLastModifiedSuffix(@Nullable String suffix) {
             String fileReference = component.getValueMap().get(DownloadResource.PN_REFERENCE, String.class);
             if (StringUtils.isNotEmpty(fileReference)) {
                 imageResource = component.getResourceResolver().getResource(fileReference);
-                source = Source.ASSET;
+                if(imageResource != null) {
```

Comment:
Please add a space between `if` and parentheses!

Reply:
```suggestion
                if (imageResource != null) {
```